docs/getting-started/quick-start-guide.asciidoc

```diff
@@ -0,0 +1,135 @@
+[[get-started]]
+== Quick start
+
+To quickly get up and running with {kib}, set up on Cloud, then add a sample data set that you can use to explore and analyze.
+
+[float]
+=== Required privileges 
```

Comment:
Instead of "Required privileges", could this be "Before you begin" with two links

- Some text about knowing what Kibana is for and linking to What is Kibana? Or is that too deep?
- Have these privileges....

Reply:
I changed this `Before your begin`, but the rule of thumb is that you don't need to link out to topics that are within 3-5 topics.